tests/tests/eth_provider.rs

```diff
@@ -283,6 +284,72 @@ async fn test_get_logs(#[future] katana: Katana, _setup: ()) {
     assert!(!logs.is_empty());
 }
 
+async fn filter_logs_and_assert(filter: Filter, expected_count: usize, provider: Arc<dyn EthereumProvider>) {
+    let logs = provider.get_logs(filter).await.expect("Failed to get logs");
+    let logs = match logs {
+        FilterChanges::Logs(logs) => logs,
+        _ => panic!("Expected logs"),
+    };
+    assert_eq!(logs.len(), expected_count);
+}
+
+#[rstest]
+#[awt]
+#[tokio::test(flavor = "multi_thread")]
+async fn test_get_logs_topics(#[future] katana: Katana, _setup: ()) {
+    // Given
+    let provider = katana.eth_provider();
+    let logs = katana.logs_with_min_topics(3);
+    let topic_one = logs[0].topics()[0];
+    let topic_two = logs[1].topics()[1];
+    let topic_three = logs[0].topics()[2];
+    let topic_four = logs[1].topics()[2];
```

Comment:
Actually it's because `topic_three` and `topic_four` are used in order to perform a union query on the third topic -> `[any, any, [topic_three OR topic_four], any]`

Reply:
can rename if you prefer

Reply:
No this is ok, juts wanted to be sure :)